src/core/insights/mixpanel.ts

```diff
@@ -6,6 +6,7 @@ export const initMixpanel = (
   token: string,
   autoCapture: boolean = false,
   debug: boolean = false,
+  recordSessionsPercent: number = 1,
```

Comment:
We don't need to explicitly state a type when using a primitively typed default value such as this

Reply:
TIL